imagenet/main.py

```diff
@@ -415,7 +415,7 @@ def update(self, val, n=1):
         self.avg = self.sum / self.count
 
     def all_reduce(self):
-        total = torch.FloatTensor([self.sum, self.count])
+        total = torch.FloatTensor([self.sum, self.count]).cuda(self.sum.device)
```

Comment:
i think it should be `total = torch.Tensor([self.sum, self.count], device=self.sum.device, dtype=torch.float32)`

Reply:
> i think it should be `total = torch.Tensor([self.sum, self.count], device=self.sum.device, dtype=torch.float32)`


That should work too :)

Reply:
Both works. I think " torch.tensor(...)" is more recommended nowadays. @JoohyungLee0106 would you like to make the change?

Reply:
@hudeven Sure.